management-interface.go

```diff
@@ -221,8 +266,22 @@ func NetworkHandler(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	// Read online/offline status from 'network.yaml'
+	config, err := ParseNetworkConfig(networkConfigFile)
+	if err != nil {
```

Comment:
If an error is returned here something has gone fairly wrong. Can it be surfaced to the user?